### DISCUSSION TITLE
2022/12/31/2022-fw-to-2023-fw/

### DISCUSSION THREAD

**Discussion by giscus** [DeCalvin2006/blog#12] [General]
# 2022/12/31/2022-fw-to-2023-fw/

新年快乐。

https://blog.decalvin.tk/2022/12/31/2022-fw-to-2023-fw/

**Comment by shiziping** [comment]
祝好，希望您能够尽快地走出人生的阴霾 QwQ

**Reply by DeCalvin2006** [reply]
怎么就阴霾了（

**Reply by shiziping** [reply]
Ummm 至少我认为是的（
